arangod/StorageEngine/EngineSelectorFeature.cpp

```diff
@@ -76,10 +76,10 @@ void EngineSelectorFeature::prepare() {
       LOG_TOPIC(FATAL, Logger::STARTUP) << "unable to read content of 'ENGINE' file '" << _engineFilePath << "': " << ex.what() << ". please make sure the file/directory is readable for the arangod process and user";
       FATAL_ERROR_EXIT();
     }
-  } else {
-    if (_engine == "auto") {
-      _engine = MMFilesEngine::EngineName;
-    }
+  }
+    
+  if (_engine == "auto") {
+    _engine = MMFilesEngine::EngineName;
   }
 
   TRI_ASSERT(_engine != "auto");
```

Comment:
pls remove this assert, its not needed anymore.

Reply:
it does not hurt either.